app/lib/pre_assembly/from_staging_location/file.rb

```diff
@@ -31,12 +31,8 @@ def initialize(file:)
 
       delegate :sha1, :md5, :provider_md5, :provider_sha1, :mimetype, :filesize, to: :file
 
-      def file_id(common_path:)
-        # set file id attribute, first check the relative_path parameter on the object, and if it is set, just use that
-        return file.relative_path if file.relative_path
-
-        # if the relative_path attribute is not set, then use the path attribute and check to see if we need to remove the common part of the path
-        common_path ? file.path.gsub(common_path, '') : file.path
+      def file_id(common_path: '')
+        file.path.gsub(common_path, '')
       end
 
```

Comment:
the relative path as we are currently defining it in `ObjectFileFinder` does not make sense to me...it always throws away the hierarchy and just leaves the filename ... so we either need to change how that works or just ignore it and return the path with the common  path removed here